localized/games/wordwindow/wordwindow_level_hi.tsv

```diff
@@ -92,16 +92,16 @@ en-US	7	2	5	2	"एक किसान ने 16 अंडे इकट्ठे
-en-US	8	3	1	5	"एक बॉक्स में 4 गेंदें हैं। उनमें से 2 सॉकर बॉल हैं और बाकी बास्गौरी बॉल हैं। बास्केटबाल कितने हैं?"	एक बॉक्स में 4 गेंदें हैं। उनमें से 2 सॉकर बॉल हैं और बाकी बास्गौरी बॉल हैं। बास्केटबाल कितने हैं?	2	4	6	8	basketballs	basketballs	basketballs	basketballs	4 = 2 + <2>	1	2	2	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
+en-US	8	3	1	5	"एक बॉक्स में 4 गेंदें हैं। उनमें से 2 सॉकर बॉल हैं और बाकी बास्केट बॉल हैं। बास्केटबाल कितने हैं?"	एक बॉक्स में 4 गेंदें हैं। उनमें से 2 सॉकर बॉल हैं और बाकी बास्केट बॉल हैं। बास्केटबाल कितने हैं?	2	4	6	8	basketballs	basketballs	basketballs	basketballs	4 = 2 + <2>	1	2	2	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
 en-US	8	3	2	5	"अदिति के पास 5 किताबें हैं। उनमें से 3 विज्ञान की किताबें हैं और बाकी स्टोरीबुक हैं। स्टोरीबुक कितने हैं?"	अदिति के पास 5 किताबें हैं। उनमें से 3 विज्ञान की किताबें हैं और बाकी स्टोरीबुक हैं। स्टोरीबुक कितने हैं?	2	3	4	5	storybooks	storybooks	storybooks	storybooks	5 = 3 + <2>	1	3	2	0	0					wordwindow-stone-yellow.png	wordwindow-stone-red.png		
-en-US	8	3	3	5	"सुमित और रेवेन ने एक साथ 8 तस्वीरें खींचीं। सुमित ने 3 तस्वीरें खींचे। रेवेन ने कितने तस्वीरें खींचे?"	सुमित और रेवेन ने एक साथ 8 तस्वीरें खींचीं। सुमित ने 3 तस्वीरें खींचे। रेवेन ने कितने तस्वीरें खींचे?	3	5	8	11	pictures	pictures	pictures	pictures	8 = 3 + <5>	2	3	5	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
+en-US	8	3	3	5	"सुमित और वैशाली ने एक साथ 8 तस्वीरें खींचीं। सुमित ने 3 तस्वीरें खींचे। वैशाली ने कितने तस्वीरें खींचे?"	सुमित और वैशाली ने एक साथ 8 तस्वीरें खींचीं। सुमित ने 3 तस्वीरें खींचे। वैशाली ने कितने तस्वीरें खींचे?	3	5	8	11	pictures	pictures	pictures	pictures	8 = 3 + <5>	2	3	5	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
 en-US	8	3	4	5	"उर्मि की अलमारी में 7 शर्ट हैं। उनमें से 3 नीले हैं और बाकी हरे हैं। उसके पास कितनी हरी शर्ट है?"	उर्मि की अलमारी में 7 शर्ट हैं। उनमें से 3 नीले हैं और बाकी हरे हैं। उसके पास कितनी हरी शर्ट है?	3	4	5	6	green shirts	green shirts	green shirts	green shirts	7 = 3 + <4>	2	3	4	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
 en-US	8	3	5	5	"सुश्री अंजलि ने चीनी के 8 बैग का आदेश दिया। 5 बैग आज वितरित किए गए हैं। बाकी कल पहुंचाए जाएंगे। कल चीनी के कितने बैग वितरित किए जाएंगे?"	सुश्री अंजलि ने चीनी के 8 बैग का आदेश दिया। 5 बैग आज वितरित किए गए हैं। बाकी कल पहुंचाए जाएंगे। कल चीनी के कितने बैग वितरित किए जाएंगे?	2	3	5	8	bags	bags	bags	bags	8 = 5 + <3>	2	5	3	0	0					wordwindow-stone-red.png	wordwindow-stone-blue.png		
 en-US	9	1	1	6	"नकुल के पास 4 मार्बल्स हैं। सू ने नकुल को कुछ और मार्बल्स दिए और अब उसके पास कुल 6 मार्बल्स हैं। सू ने नकुल को कितने मार्बल्स दिए?"	ीटर के पास 4 मार्बल्स हैं। सू ने नकुल को कुछ और मार्बल्स दिए और अब उसके पास कुल 6 मार्बल्स हैं। सू ने नकुल को कितने मार्बल्स दिए?	2	3	4	5	marbles	marbles	marbles	marbles	4 + <2>= 6	1					4	2	+	B	wordwindow-stone-red.png	wordwindow-stone-blue.png		
```

Comment:
Fix issue with Hindi word on line 107. Also, search for other instances of similar issue in entire file using the character.